packages/ui/src/theme/css/component/image.scss

```diff
@@ -3,4 +3,8 @@
   max-width: var(--amplify-components-image-max-width);
   object-fit: var(--amplify-components-image-object-fit);
   object-position: var(--amplify-components-image-object-position);
+
+  // need to set overflow to clip to resolve chrome 108 breaking change:
+  // https://developer.chrome.com/blog/overflow-replaced-elements/
+  overflow: clip;
```

Comment:
Should this be applied to all browsers or just chrome?

Reply:
Good q, i'm researching on my own right now to see if it's an acceptable change for other browsers

Reply:
Reading through https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#values:

> `clip`: [...] Similar to hidden, the content is clipped to the element's padding box. 

as opposed to default behavior where content might be rendered outside the box. I believe this is expected, so that image dimension is consistent with whatever width and height is specified.

Not a css expert though, need confirmation from primitive folks.

Reply:
@zchenwei can you help in validating across other browsers?

Reply:
And @0618

Reply:
Just confirmed on firefox that this setting does not affect the rendered result: 

https://user-images.githubusercontent.com/43682783/209990035-23b17e6d-147a-4372-9acb-09556ae89f42.mp4

Testing with safari now, but I'm good with the fix now.

Reply:
Just confirmed with Safari too

Reply:
Validating now